src/classes/STG_PanelDataImportAdvancedMapping_CTRL.cls

```diff
@@ -174,7 +174,6 @@ public with sharing class STG_PanelDataImportAdvancedMapping_CTRL extends STG_Pa
 
             } catch (Exception e) {
                 Database.rollback(sp);
-                ERR_Handler.processError(e, ERR_Handler_API.Context.STTG);
```

Comment:
What's the thinking behind removing these calls to ERR_Handler.processError() (or conversely, what was the thinking behind including them initially?)

Reply:
So the processErrors method from ERR_Handler takes in a an exception, a list of exceptions, or a list of Error__c. It's really an artifact from when I copied some of the code over from the CRLPs controller. Our error messages need the custom labels/custom label combinations instead of the exception straight up so I figured we don't really need it. Plus wherever they are in our controller we'll end up with two of the same error messages.

Reply:
Oh yeah - let's remove that, good call!